src/components/timer/timer.js

```diff
@@ -0,0 +1,29 @@
+export const time = () => {
+
+    const timer = document.createElement("div");
+    timer.setAttribute("id", "timer")
+    document.getElementById("app").appendChild(timer);
+
+
+    let timeLeft = 60;
+    let downloadTimer = setInterval(() => {
+        if (timeLeft <= 0) {
+            clearInterval(downloadTimer);
+            document.getElementById("timer").innerHTML = "00:00";
+            return;
+        }
+        timeLeft -= 1;
+
+        document.getElementById("timer").innerHTML = prettyPrint(timeLeft);
+
+    }, 1000);
+
+    function prettyPrint(timeLeft) {
+        if (timeLeft >= 10) {
+            return '00' + ':' + timeLeft;
+        } else {
+            return '00' + ':' + '0' + timeLeft;
+        }
+    }
```

Comment:
It seems to me that this function is redundant and it can be written more simply. The timer is working very nicely Congratulations.

Reply:
I can try to change it, but I read, that creating smaller classes, functions, or modules will lead to more robust and maintainable code. In addition, code becomes easier to unit test.

Reply:
try to use ternary operator.